src/optiondefs.h

```diff
@@ -575,6 +576,11 @@ static struct vimoption options[] =
 			    {(char_u *)0L, (char_u *)0L}
 #endif
 			    SCTX_INIT},
+#ifdef FEAT_QUICKFIX
+    {"chistory",    "chi",  P_NUM|P_VI_DEF,
+			    (char_u *)&p_chi, PV_NONE, did_set_chistory, NULL,
+			    {(char_u *)10L, (char_u *)0L} SCTX_INIT},
+#endif
```

Comment:
This seems not the correct way to specify options that are only available when certain features are defined. Seem the `'charconvert'` option above.